assets/interfaces.go

```diff
@@ -181,6 +182,53 @@ type Resthook interface {
 	Subscribers() []string
 }
 
+type TemplateUUID utils.UUID
+
+// Template is a message template, currently only used by WhatsApp channels
+//
+//   {
+//     "name": "revive-issue",
+//     "uuid": "14782905-81a6-4910-bc9f-93ad287b23c3",
+//     "translations": [
+//       {
+//          "language": "eng",
+//          "content": "Hi {{1}}, are you still experiencing your issue?",
+//          "channel": {
+//            "uuid": "cf26be4c-875f-4094-9e08-162c3c9dcb5b",
+//            "name": "Twilio Channel"
+//          }
+//       },
+//       {
+//          "language": "fra",
+//          "content": "Bonjour {{1}}",
+//          "channel": {
+//            "uuid": "cf26be4c-875f-4094-9e08-162c3c9dcb5b",
+//            "name": "Twilio Channel"
+//          }
+//       }
+//     ]
+//   }
+//
+// @asset template
+type Template interface {
+	UUID() TemplateUUID
+	Name() string
+	Translations() []TemplateTranslation
+}
+
+// TemplateTranslation represents a single translation for a specific template and channel
+type TemplateTranslation interface {
+	Content() string
+	Language() utils.Language
+	VariableCount() int
+	Channel() ChannelReference
+}
+
+// TemplateAssets represents our interface to get a template
```

Comment:
don't think this is used?

Reply:
Oh right, ya, started with `send_msg` having both `template` and `template_variables` attributes but then decided it made more sense to have a single attribute with everything.

Reply:
I think this interface still isn't used 

(and comment was meant for https://github.com/nyaruka/goflow/pull/596#discussion_r270184407)